tael.md

```diff
@@ -20,11 +20,37 @@ For example, a client that controls a downstream pipeline might subscribe to a t
 ### Implementing a client
 
 To join the Tael network, you need a valid client.
-A basic client that can access `Tael`, subscribe to a topic, automatically export the payload to the current OS environment, and run a shell command [has been implemented by Sam](https://github.com/SamStudio8/elan-nextflow/blob/master/bin/ipc/mqtt-client.py).
+A basic client that can access `Tael`, subscribe to a topic, automatically export the payload to the current OS environment, and run a shell command is available in the `/cephfs/covid/software/sam/public/mqtt-client.py` directory in CLIMB.
 
+This must be used from a clean conda environment which can be set up with the following commands:
+```
+$ conda create -n tael-messenger python=3.7
+$ conda activate tael-messenger
+$ pip install paho-mqtt
+```
+
+You can set up a client which will listen to a topic with the following commands:
+
+```
+$ conda activate tael-messenger
+$ python /cephfs/covid/software/sam/public/mqtt-client.py -c '<COMMAND>' -t 'COGUK/<TOPIC>/status' --who <CLIENT_NAME>
+```
+The `-c` argument should be a command which will be executed when the topic receives a message with the attribute `status finished`.
+
+---
 ### Sending a message
 
-A very simple program to send a message through Tael [has also been implemented by Sam](https://github.com/SamStudio8/elan-nextflow/blob/master/bin/ipc/mqtt-message.py).
+Send a test mesage with the following:
+
+```
+$ conda activate tael-messenger
+$ python /cephfs/covid/software/sam/public/mqtt-message.py -t 'COGUK/<TOPIC>/status' -attr date YYYYMMDD -attr elan_date YYYYMMDD --attr status finished
```

Comment:
sorry missed this the first time, you can remove the elan_date here, it's not needed for testing (neither is date), perhaps add a sentence: "You can send any key value pair you like with --attr <KEY> <VALUE>`."